docs/products/postgresql/howto/pgbouncer-stats.rst

```diff
@@ -0,0 +1,37 @@
+Accessing PGBouncer statistics
+==============================
+
+PGBouncer is used at Aiven for :doc:`../concepts/pg-connection-pooling`.
+
+Get PGBouncer URL
+------------------
```

Comment:
can we do a one-liner about looking under "Pools" in the web interface as well?

Reply:
Added